validation/src/main/scala/hmda/validation/context/ValidationContext.scala

```diff
@@ -2,4 +2,4 @@ package hmda.validation.context
 
 import hmda.model.institution.Institution
 
-case class ValidationContext(institution: Option[Institution])
+case class ValidationContext(institution: Option[Institution], filingYear: Option[Int])
```

Comment:
This line also seems like a potential site for hard-coding 2017.
Currently not my first choice, but I could imagine arguments that would sway me. And it is certainly better (more central) than the current location(s), such as line 13 of `LarQualityEngine`. (That 2017 is semantically the same year as this one, right?)